setup.py

```diff
@@ -42,10 +42,7 @@
     'wandb',
 ]
 
-test_requires = [
-    'pytest',
-    'pytest-cov',
-]
+test_requires = ['pytest', 'pytest-cov', 'torchtyping']
```

Comment:
```suggestion
test_requires = [
    'pytest', 
    'pytest-cov', 
    'torchtyping',
]
```
The typical formatting in PyG (the trailing comma tops it from becoming a list)